submit.go

```diff
@@ -118,6 +120,14 @@ type submitResponse struct {
 	ReportURL string `json:"report_url,omitempty"`
 }
 
+// regex to catch and substitute ambiguous issue references with explicit ones to the actual repo they are in
+var ambiguousIssueRegex = regexp.MustCompile(`(^|[([{\s])(#\d+)([^\w]|$)`)
```

Comment:
could you write a comment which describes what this is matching? It's quite hard to grok by eye. (though having stared at it for several minutes, I think I understand it and it looks plausible).